guidelines/2.6.1.md

```diff
@@ -0,0 +1,135 @@
+This document is in beta. Help us by [reporting issues via Github](https://github.com/theappbusiness/accessibility-guidelines) or [email](mailto:alexander.newnham@kinandcarta.com)
+
+[Back to the overview page](./../index.html)
+
+# Touch Target Size and Spacing
+
+On this page:
+- [Summary](#summary)
+- [Requirements](#requirements)
+- [Guidance for Design](#guidance-for-design)
+- [Guidance for iOS](#guidance-for-ios)
+- [Guidance for Android](#guidance-for-android)
+- [Guidance for Flutter](#guidance-for-flutter)
+- [Guidance for Web](#guidance-for-web)
+- [More info](#more-info)
+  
+---
+
+## Summary
+
+**Make sure that touch targets are big enough and have enough distance from each other.**
+
+It should be easy to interact with all the touchable items in screen.
+
+---
+
+## Requirements
+
+- Ensuring that touch targets are at least 9 mm high by 9 mm wide.
+- Ensuring that touch targets close to the minimum size are surrounded by a small amount of inactive space.
+   
+**Note**: This size is not dependent on the screen size, device or resolution. Screen magnification should not need to be used to obtain this size, because magnifying the screen often introduces the need to pan horizontally as well as vertically, which can decrease usability.
+
+
+### Common mistakes
+
+- Small clickable texts (similar to how **links** look & feel in web).
+- *(TODO: Add more examples)*
+
+### Why?
+
+Many people have difficulty focusing on small touch targets on the screen. This could be because their fingers are large, or because they have a medical condition that impairs their motor skills. Small touch targets also make it harder for screen reader users to navigate apps using explore by touch.
+
+### Official wording in the Web Content Accessibility Guidelines
+
+> [**3.3 Touch Target Size and Spacing:**](https://www.w3.org/TR/mobile-accessibility-mapping/#touch-target-size-and-spacing) The high resolution of mobile devices means that many interactive elements can be shown together on a small screen. But these elements must be big enough and have enough distance from each other so that users can safely target them by touch.
+
+
+See the [W3C's detailed explanation of this guideline](https://www.w3.org/TR/mobile-accessibility-mapping/#touch-target-size-and-spacing).
+
+---
+
+## Guidance for Design
+
+This section needs more content. [Contribute via Github](https://github.com/theappbusiness/accessibility-guidelines/) or [email](mailto:talia.craiu@kinandcarta.com).
```

Comment:
I wonder if we should be adding issues for this that we can link to in bench tickets? that way if any designers (or iOS devs etc etc) are available they can pick up a small chunk of the work?

Reply:
Sounds good. Let's discuss it in today's gathering. We could have a ticket per section (Design, iOS , Android, Web) and then add all the missing guidances as subtasks/checkboxes. That way it might be easier to assign a person to a task?